src/lib/container.js

```diff
@@ -48,6 +61,7 @@ module.exports = {
 			structureTarget = target !== '' ? `/${target}` : '';
 			configurator.startCLI(componentName => {
 				componentName = this.CaptalizeFirstLetter(componentName);
+				componentName = this.EliminateSpaces(componentName);
```

Comment:
why not use `EliminateSpaces` logic inside `CaptalizeFirstLetter` ?

Reply:
just thought it would be nice to separate them, no real purpose though